### PR TITLE
add `aks-store-demo` template

### LINKED ISSUE #289: [Idea] https://github.com/Azure-Samples/aks-store-demo
## Azure Services (**optional**)
_What is this the purpose of this template? Are there specific pieces of technology or a specific stack you're interested in?_.

OpenAI + AKS + Terraform

## Next Steps
Once submitted, this issue will be tagged with `requested-contribution` so that anyone can pick it up and create the template.

### PR DESCRIPTION
**Pull request by hemarina**
fix #289 
cc @jongio @pauldotyu for notification
Microsoft learning page support: https://github.com/Azure-Samples/aks-store-demo/issues/91

### CODE REVIEW

**Comment by v-xuto** [comment]
@hemarina Sure. We will test this template today.

**Comment by v-xuto** [comment]
@hemarina We have finished this template test, and we filed one issue. Please review.

- https://github.com/Azure-Samples/aks-store-demo/issues/92

File: website/static/templates.json [hunk 1]
```diff
@@ -1376,30 +1362,24 @@
...
-    ]
+    "source": "https://github.com/Azure-Samples/openai-aca-lb",
+    "tags": ["aca", "openai", "ai", "msft", "bicep", "dotnetCsharp", "new"]
+  },
+  {
+    "title": "Azure Kubernetes Service Store Demo",
+    "description": "Sample microservices app for AKS demos, tutorials, and experiments.",
+    "preview": "./templates/images/aks-store-demo.png",
+    "website": "https://github.com/Azure-Samples",
+    "author": "Azure Content Team",
```
Comment (pauldotyu) [inline website/static/templates.json]:
```suggestion
    "author": "AKS Content Team",
```

File: website/static/templates.json [hunk 2]
```diff
@@ -1376,30 +1362,24 @@
...
+    "source": "https://github.com/azure-samples/aks-store-demo",
+    "tags": ["terraform", "aks", "openai", "ai", "msft", "new"]
```
Comment (pauldotyu) [inline website/static/templates.json]:
```suggestion
    "tags": ["terraform", "aks", "openai", "ai", "msft", "new", "kubernetes", "helm" ]
```

**Review by pauldotyu — COMMENTED** [review]
LGTM. Just some minor author and tags updates.

File: website/src/data/tags.tsx [hunk 3]
```diff
@@ -219,6 +220,11 @@ export const Tags: { [type in TagType]: Tag } = {
     description: "Template architecture uses Web Components",
     type: "Tools",
   },
+  helm: {
+    label: "Helm",
+    description: "Template architecture uses Helm",
+    type: "Tools",
```
Comment (hemarina) [inline website/src/data/tags.tsx:226]: Add helm tags under tools. cc @pauldotyu for notification.

**Comment by v-xuto** [comment]
@hemarina After re-run it with `az login`, it works.  And there are currently no issues with this template.

**Review by ellismg — APPROVED** [review]

**Pull request merged**